### ISSUE TITLE
Copyright header is wrong

### ISSUE THREAD

**Issue by peterwvj** [overturetool/overture#636]
The copyright header has the wrong year "(Copyright (c) Overture contributors and others 2009-2016)". It should be 2017.

**Comment by nickbattle** [comment]
I guess this comes from a fixed text file? Would it be possible to automatically update it during the build - perhaps as a filtered resource with a ${year} substitution? We have got this wrong on several releases over the years (literally!).

**Comment by peterwvj** [comment]
Yes, you are right - see #602. At least this issue only occurs once every year :-) This time, let's aim for a more robust solution.
